### ISSUE TITLE
DNS in Chrome

### ISSUE THREAD

**Issue by 174n** [popura-network/Popura#29]
Chrome doesn't resolve DNS queries to the yggdrasil addresses while firefox does. I found one solution to this problem, you might consider adding this route creation at the start

>Add a network route to 2001:4860:4860::8888. Doesn't have to actually be functional (could just drop requests to it). As long as Chrome can connect a UDP socket to the address, it will pass the heuristic checking IPv6-connectivity.

https://chromium.googlesource.com/chromium/src/+/refs/heads/main/net/dns/README.md

Here is the issue from Yggdrasil's repo https://github.com/yggdrasil-network/yggdrasil-go/issues/883, I don't think they are gonna implement that

**Comment by rany2** [comment]
You could setup a dummy interface with an IP address/route to 2001:4860:4860::8888. It's pretty easy to do that with NetworkManager or ifupdown. No need for this to be in Popura...   

Also different Chrome forks use different IPv6 addresses for this check, like UGC and Brave that use RIPE's DNS root name server. I don't think it's a good idea to just keep adding network routes so IPv6 DNS works out of the box for all those browsers.

At the end of the day Chrome and the browsers based on it should stop doing this... It no longer makes sense. In case you're wondering why this "feature" was added it is because crappy routers would crash if you send an AAAA query to them.

**Comment by 174n** [comment]
I agree, Chrome should stop doing this, but I don't think Popura/Yggdrasil can influence their decision. If it's not a good idea to add it for everyone by default, I think adding that option at least temporarily to the config would greatly improve the user experience for less geeky users. The majority wouldn't even know what the problem is

**Comment by zhoreeq** [comment]
Maybe we could add an option `-fixchrome` and make it enabled by default in the "Auto" build. 

But it could create problems for users with native IPv6. We should at least check if the address is routable already.
